### PR TITLE
Initial split into separate repo

### PR DESCRIPTION
**Pull request by jamwaffles**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+categories = ["embedded", "no-std"]
+keywords = ["embedded-graphics", "simulator", "graphics", "embedded"]
+readme = "./README.md"
+license = "MIT OR Apache-2.0"
+edition = "2018"
+exclude = [
+    ".circleci",
+    ".travis.yml",
+    ".gitignore",
+    "convert_1bpp.sh",
```
Comment (rfuest) [inline Cargo.toml]:
```suggestion
    "/.circleci/",
    "/.github/",
    ".gitignore",
```

File: README.tpl [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Embedded graphics simulator
+
+[![Build Status](https://circleci.com/gh/embedded-graphics/embedded-graphics/tree/master.svg?style=shield)](https://circleci.com/gh/embedded-graphics/embedded-graphics/tree/master)
```
Comment (rfuest) [inline README.tpl]: This needs to be updated to use the simulator build status.

**Review by rfuest — CHANGES_REQUESTED** [review]
The `doc` and `examples` directories shouldn't be in this repo, but we can move them later.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+### Changed
+
+- The simulator is now [available on crates.io](https://crates.io/crates/embedded-graphics-simulator) as a standalone crate. You can now create simulated displays for testing out embedded_graphics code or showing off cool examples.
+- The builtin simulator now supports colour pixel types, like `RGB565`.
+
+<!-- next-url -->
+
+[unreleased]: https://github.com/embedded-graphics/simulator/compare/after-split...HEAD
+[unreleased - `embedded-graphics` repository]: https://github.com/embedded-graphics/embedded-graphics/compare/embedded-graphics-simulator-v0.2.0...HEAD
```
Comment (rfuest) [inline CHANGELOG.md]:
```suggestion
[unreleased - `embedded-graphics` repository]: https://github.com/embedded-graphics/embedded-graphics/compare/embedded-graphics-simulator-v0.2.0...before-split
```

**Review by rfuest — CHANGES_REQUESTED** [review]
One small nit left...

**Comment by jamwaffles** [comment]
I was so close  *shakes fist

**Review by rfuest — APPROVED** [review]
Looks good now.

**Pull request merged**